src/libraries/shims/ApiCompat.proj

```diff
@@ -26,13 +14,27 @@
     <NetFxReference Include="System.DirectoryServices.Protocols" />
   </ItemGroup>
 
+  <Target Name="SetApiCompatFiles">
+    <PropertyGroup>
```

Comment:
Nit, add a comment into why this needs to be inside a target, as we discussed before.